tensorflow_probability/python/experimental/mcmc/with_reductions_test.py

```diff
@@ -0,0 +1,499 @@
+# Copyright 2020 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for WithReductions TransitionKernel."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+
+# Dependency imports
+from absl.testing import parameterized
+import numpy as np
+
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+from tensorflow_probability.python.experimental.mcmc import CovarianceReducer
```

Comment:
I'd avoid these and just do:

tfp_exp_mcmc = tfp.experimental.mcmc # Or a different alias of your choice

You're already depending on the entirety of TFP, so might as well use the public access points. Also, as is, these violate the style guide where we are forbidden from importing symbols from modules directly (except in dire circumstances).